### PR TITLE
[12.0][FIX] ValidationError string is not formatted with %s

### PR DESCRIPTION
**Pull request by sergiocorato**
Without this PR, ValidationError raise a server error.

### CODE REVIEW

File: stock_account_inventory_force_date/models/stock_inventory.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def _get_inventory_lines_values(self):
                 lines.extend(self._generate_lines_at_date(loc, product_ids))
         else:
             raise ValidationError(_(
-                'Option %s not available when forcing Inventory Date.',
+                'Option %s not available when forcing Inventory Date.' %
```
Comment (rousseldenis) [inline stock_account_inventory_force_date/models/stock_inventory.py]: @sergiocorato  Are you sure of the error you have?

_() translation function supports arguments for string formatting like previous code.

Comment (rousseldenis) [reply stock_account_inventory_force_date/models/stock_inventory.py]: Mmmh, right, this is true for v >= 14

**Review by rousseldenis — APPROVED** [review]

File: stock_account_inventory_force_date/models/stock_inventory.py (hunk 1 above)
Comment (sergiocorato) [reply stock_account_inventory_force_date/models/stock_inventory.py]: Mmm, overriding %s python standard function?

Comment (sergiocorato) [reply stock_account_inventory_force_date/models/stock_inventory.py]: Ah, took too long to reply :)

**Comment by sergiocorato** [comment]
@rousseldenis time to merge :) ?

**Review by rousseldenis — APPROVED** [review]

**Review by kos94ok-3D — APPROVED** [review]

**Comment by sergiocorato** [comment]
> @rousseldenis your merge command was aborted due to failed check(s), which you can inspect on [this commit of 12.0-ocabot-merge-pr-1351-by-rousseldenis-bump-patch](https://github.com/OCA/stock-logistics-warehouse/commits/fbf0d6d2a73b2837070fe22c48dbcaaae88a0f0a).
> 
> After fixing the problem, you can re-issue a merge command. Please refrain from merging manually as it will most probably make the target branch red.

Module `microsoft_outlook` ?

**Comment by rousseldenis** [comment]
> > @rousseldenis your merge command was aborted due to failed check(s), which you can inspect on [this commit of 12.0-ocabot-merge-pr-1351-by-rousseldenis-bump-patch](https://github.com/OCA/stock-logistics-warehouse/commits/fbf0d6d2a73b2837070fe22c48dbcaaae88a0f0a).
> > After fixing the problem, you can re-issue a merge command. Please refrain from merging manually as it will most probably make the target branch red.
> 
> Module `microsoft_outlook` ?

FYI https://github.com/odoo/odoo/pull/87389

**Comment by pedrobaeza** [comment]
@rousseldenis OCB was still with the error till midnight sync. I have manually synced it and you can retry now.

**Comment by rousseldenis** [comment]
> @rousseldenis OCB was still with the error till midnight sync. I have manually synced it and you can retry now.

Indeed, thanks

**Pull request merged**